Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -66,7 +66,7 @@ public class NavigatorIndex {
         case missingBundleIdentifier
         
         @available(*, deprecated, renamed: "missingBundleIdentifier", message: "Use 'missingBundleIdentifier' instead. This deprecated API will be removed after 6.0 is released")
-        case missingBundleIndentifier
+        case missingBundleIdentifier
```

Comment:
This doesn't compile because there's already a `missingBundleIdentifier` case define 3 lines above this. 

Changing this case name is a source breaking change that has already been fixed above. This deprecated (misspelled) case will be removed in the future.

Reply:
Thank you! Understood.
I revert the difference of "Identifier".

https://github.com/apple/swift-docc/pull/876/commits/b681fae1f6efa3f908159fd386ffa9511d48bb1c